tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -66,9 +66,8 @@ def setup_cluster(ceph_cluster, config):
     cloud_type = config.get("cloud-type")
     build_type = config.get("build_type")
 
-    try:
-        Rpm(installer).query("cephadm-ansible")
-    except RpmError:
+    pkg = "cephadm-ansible"
+    if Rpm(installer).query(pkg) == f"package {pkg} is not installed":
```

Comment:
One suggestion here, wouldn't it be better for the query to return True/False based on whether the package is installed or not? In that case the `== f"package {pkg} is not installed"` comparison can be done in packages.py/query method itself.

so then, we can rephrase the above statement as

```
if not Rpm(installer).query(pkg):
     <package is not installed and we can proceed with installation>
```

Or if the idea is to get the package version via query command, then we can return None if not installed and package version when the package is installed. In both the approaches, the above statement will work .

A suggestion only, if you feel otherwise please ignore.

Reply:
done